### ISSUE TITLE
After call close.dialog() <body> obtained additional style "padding-right: 0"

### ISSUE THREAD

**Issue by creadone** [nakupanda/bootstrap3-dialog#317]
This behaviour breaking some layouts with fixed body paddings. Maybe is not necessary resetting right padding?

--
And many thanks for bootstrap3-dialog, this is the very practical tool!

**Comment by baconface** [comment]
I agree that bootstrap3-dialog should not affect the overall style of `body`.

**Comment by baconface** [comment]
I believe this has something to do with fixing the spacing for page scrolling?

**Comment by creadone** [comment]
Yes. But if needed then better move it into `css` or return `body` to the initial state.
